src/content/developers/docs/gas/index.md

```diff
@@ -54,7 +56,13 @@ For example if you put a gas limit of 50,000 for a simple ETH transfer, the EVM
 
 ## What is gas price? {#what-is-gas-price}
 
-Gas price refers to the amount of ether you are willing to pay for every unit of gas, and this is usually measured in 'gwei'.
+Gas price refers to the amount of ether you are willing to pay for every unit of gas, and this is usually measured in 'gwei'. 
+
+A bit of trivia: 'Gwei' is also called nanoether (or nano). This is because it's the ninth power of the fractional ETH. 
+
+The word 'gwei' itself means 'giga-wei', and it is equal to 1,000,000 wei. Wei itself (derived from the Chinese word for 'power') is the smallest unit of ETH. Wei is also the ninth power of the fractional gwei, which means it's the 18th power of the fractional ETH.
+
+While paying gas fees, you'll speak almost exclusively in gwei. 
```

Comment:
Given the new changes that have been added to this section regarding London with an expanded explanation, I feel this section may be best left as is. 

Perhaps we could cherry-pick the line about `The word 'gwei' itself means 'giga-wei', and it is equal to 1,000,000,000 wei.` and place below this sentence (which is located higher up on the page in the "What is gas?" section):

> In essence, gas fees are paid in Ethereum's native currency, ether (ETH). Gas prices are denoted in gwei, which itself is a denomination of ETH - each gwei is equal to 0.000000001 ETH (10<sup>-9</sup> ETH). For example, instead of saying that your gas costs 0.000000001 ether, you can say your gas costs 1 gwei. The word 'gwei' itself means 'giga-wei', and it is equal to 1,000,000,000 wei.

Note nine (9) zeros on that sentence as well instead of 6 in your original copy